src/index.js

```diff
@@ -67,10 +67,10 @@ class Graph extends Element {
         }
         if (this._config.readOnly) this._config.selfContainedMode = true;
         if (this._config.includeFonts) {
-            /*#if _STRIP_SCSS
-            //#else */
+            // #if _STRIP_SCSS
+            // //#else
```

Comment:
Do we need these comments?

Reply:
Yeah the JSCC rollup plugin requires the full if / else statement to work.